node/src/effect/announcements.rs

```diff
@@ -263,6 +270,13 @@ impl Display for ContractRuntimeAnnouncement {
             ContractRuntimeAnnouncement::BlockAlreadyExecuted(block) => {
                 write!(f, "block had been executed before: {}", block.hash())
             }
+            ContractRuntimeAnnouncement::StepCommitted(era_id, ..) => {
+                write!(
+                    f,
+                    "execution effect has been successfully committed in era: {}",
```

Comment:
Note that the `Display` impl for `EraId` already prints the word "era", so no need to output it again:
```suggestion
                    "step completed for {}",
```

Reply:
Applied suggestion